src/main/java/frc/robot/commands/WristCommand.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import frc.robot.subsystems.WristSubsystem;
+
+/* You should consider using the more terse Command factories API instead https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#defining-commands */
+public class WristCommand extends Command {
+
+  private final WristSubsystem wristSubsystem;
+  private final double armDesiredPosition;
+  private final double wristDesiredPosition;
+
+  /** Creates a new WristCommand. */
+  public WristCommand(WristSubsystem wristSubsystem, double armAngle, double wristAngle) {
+      this.wristSubsystem = wristSubsystem;
+      armDesiredPosition = armAngle;
+      wristDesiredPosition = wristAngle;
+      addRequirements(wristSubsystem);
+
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    wristSubsystem.setArmMotorPosition(armDesiredPosition);
+    wristSubsystem.setServoWristMotorSpeed(wristDesiredPosition);
```

Comment:
Keep the setArmMotorPosition here for sure, but you'll probably want to move the setServoWristMotorSpeed into initialize since it only needs to be run once